spec/several_assignments_spec.rb

```diff
@@ -27,7 +29,7 @@ class Test::Foo
   end
 
   context "when renamed" do
-    subject { Test::Foo.new "some foo": :BAZ }
+    subject { Test::Foo.new some_foo: :BAZ }
```

Comment:
And the same problem is here

Reply:
reverted the changes and they pass for me.  Can you approve the actions here so we can get spec output?

Reply:
Nope, I meant a different change. The original version got a space inside `"some foo"` to empasize this is not a proper Ruby method name. In your version you just switched from symbol `:some_foo` to the string `"some_foo"`, but both satisfy the metnod name requirements.

Reply:
OK I missed that. The challenge with supporting that is we can't have options like that explicitly defined in the signature. We'd have to bury that logic in unpacking the `**` arguments. Honestly I find this feature pretty surprising - I'm unclear on the real world use cases for it. Aliasing just in the constructor call when all other interactions are through the legal ruby method name passed to `as` is a little peculiar. Can you explain more?

Reply:
The particular challenge being Introspecting the dry initializer doesn't accurately return the constructor arguments `my_foo.method(:__dry_initializer_initialize__).parameters`